src/libmongoc/doc/mongoc_client_encryption_add_key_alt_name.rst

```diff
@@ -0,0 +1,37 @@
+:man_page: mongoc_client_encryption_add_key_alt_name
+
+mongoc_client_encryption_add_key_alt_name()
+===========================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+   MONGOC_EXPORT (bool)
```

Comment:
Any reason this file and `mongoc_client_encryption_remove_key_alt_name.rst` includes `MONGOC_EXPORT` but the other documentation files do not?

Reply:
No, this is a copy-paste error. Will remove `MONGOC_EXPORT` from the docs.